### DISCUSSION TITLE
Difficulty Index Input data

### DISCUSSION THREAD

**Discussion by hertneky** [dtcenter/METplus#1670] [File I/O] [answered]
I ran the example difficulty index use-case, and now I am planning to apply it to other data. The data provided for the use case was created using a save_ensemble_data.py script as documented in the DI [user script](https://metplus.readthedocs.io/en/main_v4.0/generated/model_applications/medium_range/UserScript_fcstGEFS_Difficulty_Index.html#python-embedding) at the top of the script. I don't see this script anywhere. Can this script be made available?
Tagging @lindsayrblank since she helped with this use case.

**Comment by lindsayrblank** [comment]
Hey @hertneky I'm afraid I don't know where this script would be if it's not in the parm directory that's associated with this use case. Someone who's touched this more recently will have to check.

**Reply by bikegeek** [reply]
I looked through the METcalcpy and METplotpy repositories and that script was not found.  @lindsayrblank, I thought NRL provided you with the data.

**Reply by lindsayrblank** [reply]
HI @bikegeek. If I remember correctly, there was a starter file that I used this script on to create more files. It's been a long time, so I'm not 100% positive.

**Comment by georgemccabe** [comment, marked as answer]
Hi @lindsayrblank , the comments at the top of the user script used in the use case mention that the save_ensemble_data.py script was used to generate the NPZ files that are used as input for the use case. This script is not used as part of the use case, but run as a pre-processing step to generate the data read by it. The comments also list Bill Campbell from NRL as an author. Do you happen to know his GitHub username? If not, I can send him an email to ask if he is familiar with the script.

**Reply by lindsayrblank** [reply]
Hey @georgemccabe. I don't know his GitHub user name. I remember a little more now. This script originally was one he had written and then I made some changes to it for this use case. He may still have the original version.

**Comment by hertneky** [comment]
Thanks all. I ended up looking at the example npz file and it looks straight forward in how the data is saved, so we will create our own pre-processing script to match that.
